### DISCUSSION TITLE
OpenDrift on COLAB

### DISCUSSION THREAD

**Discussion by jamespatrickmanning** [OpenDrift/opendrift#1144] [Q&A] [answered]
I was very excited to discover Opendrift this past week.

I would like to share a basic script with my colleagues using COLAB (https://colab.research.google.com/#) so that they do not have to install anything on their local machines.

However, I still need to install Opendrift into the online COLAB instance of Python.  

So, I guess this is a question primarily for COLAB users.  Has anyone done it already?  I tried various attempts with mambaforge and git clone but haven't found any successful combination.  

If anyone is interested, I can share with you my latest notebook attempts.

**Comment by jamespatrickmanning** [comment, marked as answer]
After another day of fiddling, I got it to work on COLAB!!! 

The code is [ here](https://colab.research.google.com/drive/1ZJV8uFOQhUjxS00gWzO4nwmUsJnrqgST#scrollTo=kmD6p5vjbhfO) if interested.

**Reply by gauteh** [reply]
Thanks, I requested access. Interesting!

**Reply by jamespatrickmanning** [reply]
[another one that is simpler](https://colab.research.google.com/drive/1ZJV8uFOQhUjxS00gWzO4nwmUsJnrqgST)

What is your email so I can share it?
